app/assets/javascripts/admin/addon/routes/admin-plugins-show.js

```diff
@@ -0,0 +1,16 @@
+import Route from "@ember/routing/route";
+import { inject as service } from "@ember/service";
+import { ajax } from "discourse/lib/ajax";
+import AdminPlugin from "admin/models/admin-plugin";
+
+export default class AdminPluginsShowRoute extends Route {
+  @service router;
+
+  model(params) {
+    return ajax("/admin/plugins/" + params.plugin_id + ".json").then(
```

Comment:
Any risk for injection attack here? (Not a rhetorical question.)

Reply:
Will put in a sanitize :+1: